Hymns/FavoritesView.swift

```diff
@@ -0,0 +1,17 @@
+import SwiftUI
+
+struct FavoritesView: View {
+    var body: some View {
+        NavigationView {
+            VStack {
+                Text("TODO: Favorites")
+            }.navigationBarTitle("Favorites")
+        }
+    }
+}
+
```

Comment:
Do you need the

#if DEBUG
...
#endif

I saw it in the video

(comment applies to all the files)

Reply:
Yes, from what I have been able to figure out thus far we can use them. The prievews are by default set to this in the more recent versions of swiftUI that we are using. So they automatically only run when in debug. I am not sure how to apply the #if DEBUG in the best way here though. Maybe we can get this pushed in and I'll look into this in the near future?